data/aksh-kr.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Akash Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Development Engineer",
+  "avatar": "https://github.com/aksh-kr.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/resilientbloke",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/aksh-kr",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/akaskr",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https://resilientbloke.hashnode.dev",
+      "icon": "Fahashnode"
```

Comment:
This icon doesn't seem to be available in the Linkfree docs. I think it will be better is you use `SiHashnode`

Reply:
Thank you Chinmay, 
I've changed the icon for hashnode and committed the changes.